test/TestAssets/coverlet.collector/CoverletInProcDataCollector.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Reflection;
+using System;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollector.InProcDataCollector;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.InProcDataCollector;
+
+[assembly: AssemblyKeyFile("key.snk")]
```

Comment:
What is this key file for?

Reply:
Mainly 2 reason
1) Without signed assembly I cannot compile I get an error that force me to have a strong name asm.
2) After this merge we have on backlog this https://github.com/tonerdo/coverlet/pull/566 so I'll update this key here and will amend test and alg to check also "public key token".